installation-usage.md

```diff
@@ -1,56 +1,56 @@
+  - [Set logging verbosity](#set-logging-verbosity)
+  - [Pull images from private registry](#pull-images-from-private-registry)
+  - [Configure the SELinux type](#configure-the-selinux-type)
+  - [Customise the daemon resource requirements](#customise-the-daemon-resource-requirements)
+  - [Restrict the allowed syscalls in seccomp profiles](#restrict-the-allowed-syscalls-in-seccomp-profiles)
+  - [Constrain spod scheduling](#constrain-spod-scheduling)
+  - [Enable memory optimization in spod](#enable-memory-optimization-in-spod)
+  - [Create a seccomp profile](#create-a-seccomp-profile)
+    - [Apply a seccomp profile to a pod](#apply-a-seccomp-profile-to-a-pod)
+    - [Base syscalls for a container runtime](#base-syscalls-for-a-container-runtime)
+    - [Label namespaces for binding and recording](#label-namespaces-for-binding-and-recording)
+    - [Bind workloads to profiles with ProfileBindings](#bind-workloads-to-profiles-with-profilebindings)
+    - [Record profiles from workloads with `ProfileRecordings`](#record-profiles-from-workloads-with-profilerecordings)
+      - [Log enricher based recording](#log-enricher-based-recording)
+      - [eBPF based recording](#ebpf-based-recording)
+      - [Merging per-container profile instances](#merging-per-container-profile-instances)
+      - [Disable profile recording](#disable-profile-recording)
+  - [Create a SELinux Profile](#create-a-selinux-profile)
+    - [Apply a SELinux profile to a pod](#apply-a-selinux-profile-to-a-pod)
+    - [Make a SELinux profile permissive](#make-a-selinux-profile-permissive)
+    - [Record a SELinux profile](#record-a-selinux-profile)
+  - [Restricting to a Single Namespace](#restricting-to-a-single-namespace)
+    - [Restricting to a Single Namespace with upstream deployment manifests](#restricting-to-a-single-namespace-with-upstream-deployment-manifests)
+    - [Restricting to a Single Namespace when installing using OLM](#restricting-to-a-single-namespace-when-installing-using-olm)
+  - [Using metrics](#using-metrics)
+    - [Available metrics](#available-metrics)
+    - [Automatic ServiceMonitor deployment](#automatic-servicemonitor-deployment)
+  - [Using the log enricher](#using-the-log-enricher)
+  - [Configuring webhooks](#configuring-webhooks)
+  - [Troubleshooting](#troubleshooting)
+    - [Enable CPU and memory profiling](#enable-cpu-and-memory-profiling)
+    - [Use a custom `/proc` location for nested environments like `kind`](#use-a-custom-proc-location-for-nested-environments-like-kind)
+  - [Notes on OpenShift and SCCs](#notes-on-openshift-and-sccs)
+    - [SELinux recording should allow `seLinuxContext: RunAsAny`](#selinux-recording-should-allow-selinuxcontext-runasany)
+    - [Replicating controllers and SCCs](#replicating-controllers-and-sccs)
+  - [Create an AppArmor profile](#create-an-apparmor-profile)
+    - [Apply an AppArmor profile to a pod](#apply-an-apparmor-profile-to-a-pod)
+    - [Known limitations](#known-limitations)
```

Comment:
Can we please revert those indentation changes?

Reply:
I wonder from where they came from, maybe some pre commit hook